src/visitors/components/signup_form.js

```diff
@@ -10,6 +10,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
@eliascodes at some point can you talk me through how you are using `Grid`

Reply:
Surye, Monday?